tests/rptest/scale_tests/many_partitions_test.py

```diff
@@ -43,12 +43,16 @@
 BIG_FETCH = 104857600
 
 # How much memory to assign to redpanda per partition. Redpanda will be started
-# with MIB_PER_PARTITION * PARTITIONS_PER_SHARD * CORE_COUNT memory
-DEFAULT_MIB_PER_PARTITION = 4
+# with (MIB_PER_PARTITION * PARTITIONS_PER_SHARD * CORE_COUNT) / (PARTITIONS_MEMORY_ALLOCATION_PERCENT / 100) memory
```

Comment:
Should `/ (PARTITIONS_MEMORY_ALLOCATION_PERCENT` be `* / (PARTITIONS_MEMORY_ALLOCATION_PERCENT)`?

Reply:
> * / (PARTITIONS_MEMORY_ALLOCATION_PERCENT)

Do you mean `* 100 / PARTITIONS_MEMORY_ALLOCATION_PERCENT`?

Reply:
Well I meant `* (PARTITIONS_MEMORY_ALLOCATION_PERCENT)`

Reply:
It was just wrong though, it should be `/` as I see now this is calculating _from_ the partition memory alloc % _to_ the amount RP should be started with.